src/lib/bootloader/grub2.rb

```diff
@@ -225,28 +209,15 @@ def mbr_line
     end
 
     def partition_line
-      # check for separated boot partition, use root otherwise
-      if Yast::BootStorage.separated_boot?
-        if stage1.boot_partition?
-          _(
-            "Install bootcode into /boot partition " \
-              "(<a href=\"disable_boot_boot\">do not install</a>)"
-          )
-        else
-          _(
-            "Do not install bootcode into /boot partition " \
-              "(<a href=\"enable_boot_boot\">install</a>)"
-          )
-        end
-      elsif stage1.root_partition?
+      if stage1.boot_partition?
         _(
-          "Install bootcode into \"/\" partition " \
-            "(<a href=\"disable_boot_root\">do not install</a>)"
```

Comment:
..._boot_root is still present in src/lib/bootloader/proposal_client.rb, remove it?

Reply:
right

Reply:
(done)